### DISCUSSION TITLE
[UDISE+ Program] Colour coding for PTR KPI seems to be incorrect

### DISCUSSION THREAD

**Discussion by Mohanapriya777** [Sunbird-cQube/community#829] [General]
Issue: Colour coding for PTR KPI is red for all districts of Puducherry

Expected: PTR value for districts in Puducherry is between 18-20. So expected colour coding should be green.

Screenshot attached for reference.
<img width="959" alt="screenshot" src="https://github.com/Sunbird-cQube/community/assets/130358614/c9c4b32d-971c-4662-9b41-4b4a3f6bb8fa">

**Comment by RNarmatha** [comment]
@pandutibil - Please look into this 
As per legend PTR is showing as Red,. Could you please take  a look

**Reply by pandutibil** [reply]
@Mohanapriya777 The current map report allows for either percentage or range indicators individually, but it does not support both simultaneously. Therefore, it's advisable to exclude PTR from the metric filter, but you can include it as meta information in the tooltip. We will take this feature into consideration for the upcoming release.

Thank You.

**Reply by Mohanapriya777** [reply]
Ok, Thank you

**Comment by RNarmatha** [comment]
@Mohanapriya777 - kindly confirm was this issue got fixed or need any support on this
